Bump moment and handsontable

CarolineDenis opened the pull request:
Fixes #7327

### Checklist

- [ ] Self-review the PR after opening it to make sure the changes look good and
      self-explanatory (or properly documented)
- [ ] Add relevant issue to release milestone

### Testing instructions

- Proceed with random workbench and batch edit tests

CarolineDenis committed:
Bump moment and handsontable

Fixes #7327

CarolineDenis committed:
Run npm i

CarolineDenis committed:
Merge branch 'main' into issue-7327

CarolineDenis committed:
fix: update beforeChange hook to support Handsontable 12.4.0 (null changes)

CarolineDenis committed:
[Fix]: Update beforeChange types in hooks to adhere new version

emenslin reviewed (changes requested):
Workbench and batch edit data sets do not open properly 

https://github.com/user-attachments/assets/edd39cda-98b9-42dd-949a-c23907c54f7d

Gitesh307 committed:
Update hooks.ts

Gitesh307 committed:
Lint code with ESLint and Prettier

Triggered by 217ad37e04c3cc9f3a932da9d8d2f809e8945562 on branch refs/heads/issue-7327

Gitesh307 committed:
Update hooks.ts
